include/circt/Analysis/FIRRTLInstanceInfo.h

```diff
@@ -77,10 +77,10 @@ class InstanceInfo {
   /// Information about a circuit
   struct CircuitAttributes {
     /// The design-under-test if one is defined.
-    igraph::InstanceGraphNode *dutNode;
+    igraph::ModuleOpInterface dutNode;
```

Comment:
nit: rename to drop "Node" suffix?

Reply:
SGTM. Changed (and changed all the debug code) in fc50801c3